mne/tests/test_epochs.py

```diff
@@ -3181,7 +3282,9 @@ def test_drop_epochs():
     # Bound checks
     pytest.raises(IndexError, epochs.drop, [len(epochs.events)])
     pytest.raises(IndexError, epochs.drop, [-len(epochs.events) - 1])
-    pytest.raises(ValueError, epochs.drop, [[1, 2], [3, 4]])
+    pytest.raises(TypeError, epochs.drop, [[1, 2], [3, 4]])
```

Comment:
Since you're touching this one anyway, can you modernize it with `with pytest.raises(..., match=...)` style? Feel free to update the others if you're up for it (or not!)